cli/src/main/java/com/devonfw/tools/ide/commandlet/BuildCommandlet.java

```diff
@@ -0,0 +1,73 @@
+package com.devonfw.tools.ide.commandlet;
+
+import com.devonfw.tools.ide.context.IdeContext;
+import com.devonfw.tools.ide.property.PathProperty;
+import com.devonfw.tools.ide.tool.ToolCommandlet;
+import com.devonfw.tools.ide.tool.gradle.Gradle;
+import com.devonfw.tools.ide.tool.mvn.Mvn;
+import com.devonfw.tools.ide.tool.npm.Npm;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * Build tool {@link Commandlet} for automatically detecting build configuration files and running the respective tool.
+ */
+public class BuildCommandlet extends Commandlet {
+
+  /**
+   * The path to build from.
+   */
+  public PathProperty path;
+
+  /**
+   * The constructor.
+   *
+   * @param context the {@link IdeContext}.
+   */
+  public BuildCommandlet(IdeContext context) {
+
+    super(context);
+    addKeyword(getName());
+    this.path = add(new PathProperty("", false, "path", true));
```

Comment:
```
$ devon help build
...
Run build job.
The build tool is auto-detected from according configuration files:
 pom.xml              maven (mvnw if present, mvn otherwise)
 build.gradle         gradle (gradlew if present, otherwise gradle)
 package.json         npm

Arguments:
                      run the default build if no arguments are given. Needs to succeed for definition-of-done.
 «args»               run the build with the given arguments.
...
```
See also
https://github.com/devonfw/ide/blob/cdf4af5de21489f3fdcbfdab41af858822e0d598/scripts/src/main/resources/scripts/command/build#L39

So this is how it behaves in devonfw-ide:
```
$ devon build clean
Running: /d/projects/demo/software/mvn/bin/mvn clean
[INFO] Scanning for projects...
...
```
In other words here I would expect a multi-valued `StringProperty` (or before #344 is implemented a `StringListProperty`) instead of a `PathProperty`.

Reply:
I've change the Property to a multivalued `StringProperty` now.